contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,317 @@
+featuredImageType: full
+author:
+  - lior-neu-ner
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+Also known as split testing or [multivariate testing](/product-engineers/what-is-multivariate-testing-examples), A/B testing enables you to measure the impact of product changes on your mobile apps.
+
+There are many A/B testing tools for mobile apps out there, but each one is built for a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
+
+## 1. PostHog
+
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Best for:** Engineering and product teams
+  
+![PostHog - best ab testing tools](../images/blog/best-mobile-app-ab-testing-tools/posthog.png)
+
+### What is PostHog?
+
+[PostHog](https://posthog.com) (that's us!) is an all-in-one platform built to help engineers create better products. It includes A/B testing **and** a whole bunch more, such as [product analytics](/product-analytics), [session replays](/session-replay), [feature flags](/feature-flags), and [surveys](/survey).
+
+It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.
+
+### Which platforms does PostHog support?
+
+- **iOS:** <span className="text-green text-lg">✔</span>
+- **Android:** <span className="text-green text-lg">✔</span>
+- **React Native:** <span className="text-green text-lg">✔</span>
+- **Flutter:** Product analytics and feature flags only
+- **Web:** <span className="text-green text-lg">✔</span>
+- **Server:** <span className="text-green text-lg">✔</span>
+
+### How much does PostHog cost?
+
+PostHog has [transparent pricing]((/pricing)) based on the usage. It’s free to get started and completely free for the first 1 million A/B testing requests. After this free monthly allowance, you'll pay $0.0001/request, and PostHog charges progressively less the more you use. You can also set billing limits to ensure you don't get surprise bills. 
+
+While PostHog is open source, its A/B testing features aren't included in the open source release.
+
+> **Further reading:** New to A/B testing? Read [a software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers) and our guide to [common A/B testing mistakes](/blog/ab-testing-mistakes).
+
+## 2. VWO
+
+- **Free tier:** <span className="text-red text-lg">✖</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
```

Comment:
Only VWO SDKs are open source, not the whole platform, so its a bit of a stretch to say the whole thing is open source.